### ISSUE TITLE
Version Fiasco!

### ISSUE THREAD

**Issue by slifty** [PCNI/OpenHMIS#36]
Our version numbers are kind of out of control.  The repo has the following versions listed depending on where you look:

0.3.0 (branch version)
1.0.0 (version listed in application.properties)
0.0.1 (this may just be local to me...)
v3 (version in the URL)
Version 2 (version in the repo description)

Proposed resolutions:
- I'd like to see the application.properties field automatically updated OR removed until we get to the point where we figure out an automated solution
- Dev versions should be merged back to master (since I think it's master) and tagged with 3.0.0 since we are not actually in 0.*
- URL stays the same
- repo description removes any mention of version.

**Comment by kfogel** [comment]
I've started working on this, based on the plan @slifty originally proposed in [this thread](https://groups.google.com/a/openhmis.org/forum/#!topic/openhmis-discuss/NaG69KxdKiE).  So far I have done the following:
- Merged `feature-compass_schema` into `0.3.0`.
- Merged `0.3.0` into `master`, resolving a trivial conflict in API.md.
- Tagged resultant commit 902e18cc5 as `3.0.0`.
- Created new milestone branch `3.1.0`.

We still need to archive these branches so they don't distract us (typically I do that by putting them in the `archive/` tag namespace, but will look around to see if any better conventions have arisen):
- `Oauth2.0.1`
- `Oauth2.0.1---Ashaar`
- `0.2.1`
- `feature-heroku`
- `feature-logs`
- `demo-client`
- `feature-errors`

Once the above is done I'll see if anything remains in this issue, but I think that will address most of it.  Possibly we should either update the version number in `src/main/resources/application.properties.example` to "3.1.0" or we should stop maintaining a version number there altogether.

**Comment by kfogel** [comment]
Okay, the `application.properties` thing is now tracked in issue #40, and all those old branches have been moved to archival tags:

```
0.2.1                   -->  archive/0.2.1
Oauth2.0.1              -->  archive/Oauth2.0.1
Oauth2.0.1---Ashaar     -->  archive/Oauth2.0.1---Ashaar
demo-client             -->  archive/demo-client
feature-compass_schema  -->  archive/feature-compass_schema
feature-errors          -->  archive/feature-errors
feature-heroku          -->  archive/feature-heroku
feature-logs            -->  archive/feature-logs
```

Closing this issue.
